packages/instant/src/components/animations/slide_up_and_down_animation.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+
+import { keyframes, styled } from '../../style/theme';
+
+const slideKeyframeGenerator = (fromY: string, toY: string) => keyframes`
+    from {
+        position: relative;
+        top: ${fromY}
```

Comment:
No semi-colon required? Think we should at least agree on convention here. Think we've always been putting them, same as with trailing commas in objects, arrays, etc...

Reply:
This is actually my biggest issue with interpolated string literals, they don't work with `prettier`, aren't type-safe and don't throw intelligible errors when you mistype something (e.g `backgroundColor` instead of `background-color`). Thoughts?

Reply:
If you use VSCode, this helps a bit: https://marketplace.visualstudio.com/items?itemName=jpoissonnier.vscode-styled-components